api/.env.example

```
@@ -1,2 +1,7 @@
 # This is a sample .env file that can be used for local development.
-ORIGINS="http://localhost http://localhost:3000 http://127.0.0.1 http://127.0.0.1:3000"
\ No newline at end of file
+ORIGINS="http://localhost http://localhost:3000 http://127.0.0.1 http://127.0.0.1:3000"
+POSTGRES_HOST="localhost"
+POSTGRES_USER="epr"
+POSTGRES_PASSWORD="epr"
+POSTGRES_DATABASE="epr_db"
```

Can we just call the database epr? Having a database called db seems redundant.
```suggestion
POSTGRES_DATABASE="epr"
```

---

Updated it.